### DISCUSSION TITLE
Remove auto log out

### DISCUSSION THREAD

**Discussion by Whydidthechickencrosstheroad** [monkeytypegame/monkeytype#5451] [Ideas]
Why does this exist? This should be removed and this is time consuming (when I was auto logged out)

**Comment by Warrobot10** [comment]
If you select remember me while signing in, you should stay logged in for around a week if you aren’t active, but if you play every so often then you shouldn’t really ever be logged out.

This auto log out exists because if you are signed in on a computer that other people may use (public computer etc.), then they will also be able to use their account, unless they log out of monkeytype, which is a security risk. This gives such people the option so the browser does not log a cookie and will basically sign them out when they close the tab.

Thanks, (I know I yapped but 🤷)

**Reply by Whydidthechickencrosstheroad** [reply]
I did press "Remember me" but it still didn't work even though, I come in there everyday

**Reply by Warrobot10** [reply]
If you clicked it, I think you might of turned remember me off, as it is ticked by default. Otherwise,  I don’t really know what’s going on.

**Reply by Whydidthechickencrosstheroad** [reply]
It's not off, it is on. I'm not sure why it is still doing that

**Reply by Warrobot10** [reply]
Maybe make an issue here on the GitHub, then devs can look into it.
